ChangeLog

```diff
@@ -4,9 +4,15 @@ Changes in squid-6.4 (15 Oct 2023):
 	- Regression: Do not use static initialization to register modules
 	- Bug 5301: cachemgr.cgi not showing new manager interface URLs
 	- Bug 5300: cachemgr.cgi assertion
+	- Fix handling of large HTTP headers in storeClientCopy()
+	- Fix store_client caller leak on certain errors
+	- Fix URL percent-encoding
```

Comment:
AFAICT, as of d189b5a, this change is not in v6 yet. Please wait before merging this PR (and label it with S-waiting-for-PR) OR remove this line.

```suggestion
```